### DISCUSSION TITLE
Base-2, Range-Encoded, Bit-sliced Indexes

### DISCUSSION THREAD

**Discussion by mskyttner** [duckdb/duckdb#5661] [Ideas]
This [link](https://www.featurebase.com/blog/range-encoded-bitmaps) describes a technique for using "bitmaps" and bitwise operations to perform fast range queries. Is this technique something that duckdb already supports or could benefit from?

**Comment by l1t1** [comment]
maybe this one ( https://clickhouse.com/docs/en/sql-reference/functions/bitmap-functions/ )  is useful
